src/main/resources/db/migration/V3__.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "user" RENAME TO "soli_users";
+
+ALTER SEQUENCE "user_seq" RENAME TO soli_users_seq;
+
+INSERT INTO room (id) VALUES (1);
```

Comment:
Jannik had a point when he recommended a table prefix.
Apparently, Postgres dislikes the name "users" for some reason, so this changes it.